### ISSUE TITLE
Number of training steps

### ISSUE THREAD

**Issue by montaser85** [roomylee/rnn-text-classification-tf#3]
How can I define the number of training steps? In my attempt, the code has stopped running after 16000 steps and a final accuracy is shown. how can I increase or decrease these steps? 
![accuracyvanilla](https://user-images.githubusercontent.com/20108007/43367203-53ebeefc-936b-11e8-9ae5-dec4f21cce9c.png)

**Comment by roomylee** [comment]
@montaser85 
total training steps are changed by `batch_size` and `num_epochs`
These are calculated by the below equation.
`total steps = num_epochs * (size of dataset / batch_size)`

https://github.com/roomylee/rnn-text-classification-tf/blob/35a8a48d163a75f2c3bc3bfd1c42e3999c866e95/train.py#L28
